providers/aws/components/ecs/setup.ftl

```diff
@@ -579,7 +579,7 @@
                     "       || return $?",
                     "       create_ecs_capacity_provider" +
                     "       \"" + region + "\" " +
-                    "       \"$\{asgName}\" " +
+                    "       \"" + ecsOnDemandCapacityProviderId + "\" " +
```

Comment:
I ended up making it the asgname as the capacity providers can’t be updated or deleted at the moment
So if the asg changes then using the same Id doesn’t work

I left the Id field there for when aws add capacity providers to CloudFormation

Probably better to remove it to make it more obvious why I did it this way

Reply:
@roleyfoley , I rolled back the changes and added a comment